.github/workflows/swift.yml

```diff
@@ -93,7 +98,8 @@ jobs:
 
       - name: Build on Ubuntu 20.04 with Swift 5.3
         run: |
-          swift build
+          swift test -c release
+          swift build -c release
           sudo ./install_ubuntu_deps.sh
           curl https://get.wasmer.io -sSfL | sh
           source /home/runner/.wasmer/wasmer.sh
```

Comment:
`.build/debug` needs to be changed below to `.build/release`, I can't add suggestions there unfortunately